reana/reana_dev/kind.py

```diff
@@ -32,13 +32,19 @@ def kind_commands():
     default=["CLUSTER"],
     help="Which components? [name|CLUSTER]",
 )
+@click.option(
+    "--node",
+    "-n",
+    multiple=True,
+    help="Which nodes to load the images to? [kubectl get nodes to see available ones]",
```

Comment:
What do you think about wrapping `kubectl get nodes` between quotes? To differentiate a bit from the rest of the text.